CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.1 (2019-01-17)
+
+- Fixed infinite scroll not working after being re-enabled. [#15](https://github.com/blackbaud/skyux-lists/pull/15)
```

Comment:
"Fixed thing to..."

Something to the effect of...?

"Fixed infinite scroll component to function properly after being re-enabled."

Reply:
Updated.